bittensor/utils/balance.py

```diff
@@ -68,7 +68,9 @@ def __init__(self, balance: Union[int, float]):
             # Assume tao value for the float
             self.rao = int(balance * pow(10, 9))
         else:
-            raise TypeError("balance must be an int (rao) or a float (tao)")
+            raise TypeError(
+                f"Balance must be an int (rao) or a float (tao). You passed: `{type(balance)}`."
```

Comment:
```suggestion
                f"Balance must be an int (rao) or a float (tao), not  `{type(balance)}`."
```
This is a nitpick, but just conforms to standards a bit better. Not necessary change if you feel otherwise.